src/clusterfuzz/_internal/base/utils.py

```diff
@@ -673,16 +675,34 @@ def restart_machine():
     os.system('sudo shutdown -r now')
 
 
-def search_bytes_in_file(search_bytes, file_handle):
+def search_bytes_in_file(search_bytes,
+                         file_handle,
+                         buffer_length=DEFAULT_SEARCH_BUFFER_LENGTH):
   """Helper to search for bytes in a large binary file without memory
   issues.
   """
```

Comment:
Let's document at least `buffer_length`, though we might as well document the whole function while we're at it.